repository/src/main/java/org/apache/atlas/tasks/TaskManagement.java

```diff
@@ -208,7 +208,11 @@ public List<AtlasTask> getByGuidsES(List<String> guids) throws AtlasBaseExceptio
     }
 
     public List<AtlasTask> getInProgressTasks() {
-        return registry.getInProgressTasks();
+        if(AtlasConfiguration.TASKS_REQUEUE_GRAPH_QUERY.getBoolean()) {
```

Comment:
I suggest using a different new property here, default should be ES query similar to TASKS_REQUEUE_GRAPH_QUERY